src/backend/distributed/sql/udfs/citus_calculate_gpid/latest.sql

```diff
@@ -0,0 +1,9 @@
+CREATE FUNCTION pg_catalog.citus_calculate_gpid(nodeid integer,
+                                                pid integer)
+    RETURNS BIGINT
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME',$$citus_calculate_gpid$$;
+COMMENT ON FUNCTION pg_catalog.citus_calculate_gpid(nodeid integer, pid integer)
+    IS 'activate a node which is in the cluster';
+
+GRANT EXECUTE ON FUNCTION pg_catalog.citus_calculate_gpid(integer, integer) TO PUBLIC;
```

Comment:
Do we also have a way to get the gpid of the current backend? i.e. the equivalent of `pg_backend_pid()`, so e.g. `citus_backend_gpid()`?

Reply:
Nope, we intended to have and and forgot. 

I can also get that and use it in the tests